Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -618,163 +559,123 @@ public List<ProcessDTO> findByMetadata(Map<String, String> metadata, boolean exa
+     * functionality must be provided in countResults() and loadData().)
      */
-    public QueryBuilder getQueryProjectTitle(String title) {
-        return createSimpleQuery(ProcessTypeField.PROJECT_TITLE.getKey(), title, true, Operator.AND);
+    @Deprecated
+    public List<Process> findByAnything(String searchQuery) throws DataException {
+        // TODO delete method stub
+        throw new UnsupportedOperationException("no longer used function");
     }
 
     /**
-     * Get query for find process by project id.
+     * Determines all processes with a specific docket.
      *
-     * @param projectId
-     *            as Integer
-     * @return QueryBuilder object
-     */
-    public QueryBuilder getQueryProjectId(Integer projectId) {
-        return createSimpleQuery(ProcessTypeField.PROJECT_ID.getKey(), projectId.toString(), true, Operator.AND);
-    }
-
-    /**
-     * Find processes by docket id.
+     * <!-- Used in DocketForm to find out whether a docket is used in a
+     * process. (Then it may not be deleted.) Is only checked for isEmpty(). -->
      *
      * @param docketId
-     *            id of docket for search
-     * @return list of JSON objects with processes for specific docket id
+     *            record number of the docket
+     * @return list that is not empty if something was found, otherwise empty
+     *         list
      */
-    public List<Map<String, Object>> findByDocket(int docketId) throws DataException {
-        QueryBuilder query = createSimpleQuery(ProcessTypeField.DOCKET.getKey(), docketId, true);
-        return findDocuments(query);
+    public Collection<?> findByDocket(int docketId) throws DAOException {
+        BeanQuery query = new BeanQuery(Process.class);
+        query.addIntegerRestriction("docket.id", docketId);
+        return getByQuery(query.formQueryForAll(), query.getQueryParameters(), 0, 1);
     }
 
     /**
-     * Find processes by template id.
+     * Determines all processes with a specific production template.
+     *
+     * <!-- Used in TemplateForm to find out whether a production template is
+     * used in a process. (Then it may not be deleted.) Is only checked for
+     * isEmpty(). -->
      *
      * @param templateId
-     *          id of template for search
-     * @return list of JSON objects with processes for specific template id
-     * @throws DataException if documents cannot be retrieved
+     *            record number of the production template
+     * @return list that is not empty if something was found, otherwise empty
+     *         list
+     * @throws DAOException
+     *             if an error occurred during the search
      */
-    public List<Map<String, Object>> findByTemplate(int templateId) throws DataException {
-        QueryBuilder query = createSimpleQuery(ProcessTypeField.TEMPLATE_ID.getKey(), templateId, true);
-        return findDocuments(query);
+    public Collection<?> findByTemplate(int templateId) throws DAOException {
```

Comment:
Is this method still in use (according to my IDEA not)? If not it can be removed.